Automattic/wp-calypso
# Move Jetpack Scan Section

#### Changes proposed in this Pull Request

* Move Jetpack Scan Section into my-sites

#### Testing instructions

Navigate the scan section on Jetpack Cloud and Calypso ( with `?flags=jetpack/features-section` ) and verify nothing has changed vs before this PR
